tests/view/position.js

```diff
@@ -519,4 +519,69 @@ describe( 'Position', () => {
 			document.destroy();
 		} );
 	} );
+
+	describe( 'getCommonAncestor()', () => {
```

Comment:
There's just a single test with positions in text nodes and it uses a `0` offset. Please add more,  with better offsets (ones which have a chance to break something).